ports/kf5codecs/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO KDE/kcodecs
+    REF v5.81.0
+    SHA512 65db534f32b81b8b10840233981e8a790a8fe17ddb9e64304f2ea8b817b4125ae737574a7235aa20b8b3766bd4c59bb8fb49dafa19a8a89160ab1e06d7d333b5
+    HEAD_REF master
+)
+
+vcpkg_find_acquire_program(GPERF)
+get_filename_component(GPERF_EXE_PATH ${GPERF} DIRECTORY)
+vcpkg_add_to_path(${GPERF_EXE_PATH})
+
+vcpkg_configure_cmake(
```

```suggestion
vcpkg_cmake_configure(
```
Please use the new functions for new ports.